docs/python.md

```diff
@@ -372,13 +342,31 @@ for i in range(4):
     pass
 """
 explorer = Node(for_str)
-explorer.find_for_loops()[0].find_for_iter().is_equivalent("enumerate(dict)")
-explorer.find_for_loops()[1].find_for_iter().is_equivalent("range(4)")
+explorer.find_for_loops()[0].find_for_iter().is_equivalent("enumerate(dict)") # True
+explorer.find_for_loops()[1].find_for_iter().is_equivalent("range(4)") # True
 ```
 
-#### `find_for_bodies`
+#### `find_bodies`
 
 ```python
+if_str = """
+if True:
+  x = 1
+"""
+explorer1 = Node(if_str)
+explorer1.find_ifs()[0].find_bodies()[0].is_equivalent("x = 1") # True
+
+while_str = """
+while True:
+  x += 1
+
+while False:
+  pass
```

Comment:
I think it might be helpful to have a `for:.. else:..` or `while:.. else:..` example, instead. Just for people with a JS background who might not realise that a `while` can have an `else` (ahem). For those poor folks, it might seem weird that `for` or `while` could have more than one body.